warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLargeFieldedListsVisitor.java

```diff
@@ -108,17 +116,26 @@ public Object visit(ASTOrNode node, Object data) {
         fields.addAll(rangeNodesByField.keySet());
         
         for (String field : fields) {
+            // if fields is not specified or the current field is in fields it can be reduced
+            boolean canReduce = (this.fields == null || this.fields.contains(field));
             
             Collection<JexlNode> eqNodes = eqNodesByField.get(field);
             Collection<JexlNode> rangeNodes = rangeNodesByField.get(field);
             
             // if "_ANYFIELD_" or "_NOFIELD_", then simply add the subset back into the children list
             // if past our threshold, then add a ExceededValueThresholdMarker with an OR of this subset to the children list
-            if (!Constants.ANY_FIELD.equals(field)
-                            && !Constants.NO_FIELD.equals(field)
-                            && (eqNodes.size() >= config.getMaxOrExpansionFstThreshold() || eqNodes.size() >= config.getMaxOrExpansionThreshold() || rangeNodes
-                                            .size() >= config.getMaxOrRangeThreshold()) && isIndexed(field)) {
-                log.info("Pushing down large (" + eqNodes.size() + ") fielded list for " + field);
+            // @formatter:off
+            if (canReduce &&
+                    !Constants.ANY_FIELD.equals(field) &&
+                    !Constants.NO_FIELD.equals(field) &&
+                    (eqNodes.size() >= config.getMaxOrExpansionFstThreshold() ||
+                            eqNodes.size() >= config.getMaxOrExpansionThreshold() ||
+                            rangeNodes.size() >= config.getMaxOrRangeThreshold()
+                    ) &&
+                    isIndexed(field)) {
+                // @formatter:on
```

Comment:
This is a personal preference thing, and I certainly won't reject the PR for this, but my preference would be to rewrite this chunk as follows:
```suggestion
            Collection<JexlNode> eqNodes = eqNodesByField.get(field);
            Collection<JexlNode> rangeNodes = rangeNodesByField.get(field);
            
            // if fields is set and the current field is not in fields, then don't add an ExceededValueThresholdMarker
            // if "_ANYFIELD_" or "_NOFIELD_", then simply add the subset back into the children list
            // if past our threshold, then add a ExceededValueThresholdMarker with an OR of this subset to the children list
            // @formatter:off
            if ((this.fields == null || this.fields.contains(field)) &&
                !Constants.ANY_FIELD.equals(field) &&
                !Constants.NO_FIELD.equals(field) &&
                (eqNodes.size() >= config.getMaxOrExpansionFstThreshold() ||
                 eqNodes.size() >= config.getMaxOrExpansionThreshold() ||
                 rangeNodes.size() >= config.getMaxOrRangeThreshold()) &&
                isIndexed(field)) {
                // @formatter:on
```

I think this makes it easier to see what all of the conditions are at a glance, instead of having to scroll up to see what `canReduce` is set to.